### PR TITLE
docs: npm help json/global command on windows

### LINKED ISSUE #7374: [BUG] `npm help json` errors on Windows
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

On Linux, `npm help json` displays correctly `man/man5/npm-json`. 

On Windows, `npm help json` attempts to display the corresponding HTML page which leads to the following error:

```bash
PS C:\GitHub\npm\cli> npm help json  
npm ERR! code 1
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c start "" file:///C:^%5Cnvm^%5Cv20.11.1^%5Cnode_modules^%5Cnpm^%5Cdocs^%5Coutput^%5Cconfiguring-npm^%5Cnpm-json.html
npm ERR! The system cannot find the file file:///C:%5Cnvm%5Cv20.11.1%5Cnode_modules%5Cnpm%5Cdocs%5Coutput%5Cconfiguring-npm%5Cnpm-json.html.
```

`npm help global` has the same problem.

### Expected Behavior

`npm help json` and `npm help global` should display the same documentation on Windows as on Linux

### Steps To Reproduce

1. Any Windows version, npm 8 or higher 
2. Run `npm help json`
3. See error

Cause:

The build process duplicates two man pages (`package-json` -> `npm-json`, `folders` -> `npm-global`). However, it does not duplicate the `html` pages. Therefore, `npm help json` won't find any HTML version of the duplicated man pages:

https://github.com/npm/cli/blob/9622597399ec93224fddf90a9209a98dbcfd6b2f/docs/lib/build.js#L96-L118

man:

image

html:

image

Possible solution:

- Duplicate also the HTML (might as well duplicate the Markdown) 

### Environment

- npm: 10
- Node.js: 20
- OS Name: Windows 10

### PR DESCRIPTION
**Pull request by klm-turing**
On Windows, `npm help json` and `npm help global` attempt to display `npm-json.html` and `npm-global.html` files which leads to an error because these files have not been generated.

This pull request duplicates the `package-json.md` and `folders.md` files in order to generate these files and make the above-mentioned commands functional.
## References
Closes: https://github.com/npm/cli/issues/7374

### CODE REVIEW

**Comment by klm-turing** [comment]
@wraithgar I had to update `nav.yml` file in order to pass `check-nav.js` test since I've also duplicated `md` files. Let me know if it is OK. Thank you

File: docs/lib/content/nav.yml [hunk 1]
```diff
@@ -224,6 +227,9 @@
   - title: package.json
     url: /configuring-npm/package-json
     description: Specifics of npm's package.json handling
+  - title: package.json
```
Comment (klm-turing) [inline docs/lib/content/nav.yml]: Please should we change this title to `npm-json` and so for `npm-global` ?

**Comment by lukekarrys** [comment]
Thanks for your work on this! I had to double-check but we don't actually want to add the aliases for the markdown docs. The markdown files are used in https://github.com/npm/documentation and get their own html redirects in that repo. It's not ideal to have these separate but there are many more html redirects in order to prevent breaking old urls, and those will never apply to the man or html pages (which are just used as a man page replacement on Windows).

**Comment by klm-turing** [comment]
You are welcome @lukekarrys. Thanks for your inputs and comments.

**Review by lukekarrys — APPROVED** [review]

**Comment by mbtools** [comment]
thanks @klm-turing 👍

**Pull request merged**